pkg/util/validation/field/path.go

```diff
@@ -67,9 +67,14 @@ func (p *Path) Key(key string) *Path {
 
 // String produces a string representation of the Path.
 func (p *Path) String() string {
+	return p.StringRelative(nil)
+}
+
+// String produces a string representation of the Path.
```

Comment:
It's unclear to me which direction this goes.  An example or two of inputs and results would help a lot.  Also detail corner cases if the args are wrong.